a.js

```diff
@@ -0,0 +1,9 @@
+const humberger = document.querySelector('.humberger');
+const ulnav = document.querySelector('.ulnav');
+const logo = document.querySelector('.aa');
+
+humberger.addEventListener('click', () => {
+  logo.classList.toggle('none');
+  ulnav.classList.toggle('active');
+  humberger.classList.toggle('active');
+});
```

Comment:
- [ ] One of the project requirements is when one of the links is clicked from the menu it should go to the appropriate part of the page which you implemented nicely but the mobile menu should disappear which you didn't, Kindly fix that.

![image](https://user-images.githubusercontent.com/59416136/167709053-bf272d9c-a39d-49e8-97bd-7c867e10ed46.png)